src/audio/endpoint.rs

```diff
@@ -0,0 +1,25 @@
+use rodio::Endpoint as REndpoint;
+use rodio::*;
+
+/// A speaker(s) through which audio can be played.
+pub struct Endpoint {
```

Comment:
Could we give it another name? I think something like `Output` or `OutputDevice` is probably clearer to people who didn't deal with `cpal` stuff before.